no_mutations.md

```diff
@@ -11,20 +11,266 @@ kernelspec:
   name: python3
 ---
 
+```{currentmodule} tskit
+```
+
 (sec_tskit_no_mutations)=
 
-# _Do you really need mutations?_
-% remove underscores in title when tutorial is complete or near-complete
+# Do you really need mutations?
+
+In tree sequences, the genetic genealogy exists independently of the mutations that
+generate genetic variation. This tutorial aims to illustrate
+cases where the genealogy on its own may be sufficent; a number of
+analyses do not actually require genetic variation.
+
+This applies particularly to cases where you generate tree sequences by simulation.
+In this case, although simulating variable sites allows you to generate actual genetic
+data, it's worth considering the following points:
+
+1. Neutral mutations and sites can be added to a genealogy later, if you find they *are*
+needed
+2. Simulating sites and mutations increases memory requirements and takes marginally
+more CPU time
+3. Storing mutations increases the tree sequence file size
```

Comment:
```suggestion
2. Simulating sites and mutations increases memory requirements and tree sequence file size somewhat
```
why not just leave it at this? Or, could add "and CPU time (although usually this is inconsequential)"

Reply:
I think the CPU time thing might be worth adding because of e.g. https://github.com/tskit-dev/msprime/discussions/1687

Reply:
Yes, I saw that, and was trying to write it in, but the sentence wasn't being very natural and leaving out CPU doesn't imply that CPU is never a problem. I vote against leaving in "takes marginally more CPU time" since that isn't a good reason; I think the good reason is that "in some cases it takes significantly more CPU time", but that was seeming too long. Up to you?

Reply:
never mind, I see you put in something to this effect.